cpython-windows/build.py

```diff
@@ -614,230 +614,6 @@ def hack_project_files(
         pass
 
 
-PYPORT_EXPORT_SEARCH_39 = b"""
```

Comment:
The use of these constants were removed in https://github.com/indygreg/python-build-standalone/issues/342.